doc/developers/advanced_installation.rst

```diff
@@ -107,7 +119,8 @@ feature, code or documentation improvement).
       (before cythonization) will force the build to fail if OpenMP is not
       supported.
 
-#. Build the project with pip:
+#. Build the project with pip,
+   whether you are in `conda environment` or in dedicated `virtualenv`:
```

Comment:
I would not change that. This is the next step after creating and activating the env. I don't see how it could be ambiguous here. I'm probably biased but reading this makes me wonder "why is there a need for this precision ? Is there something I missed ?".